common/config/config.go

```diff
@@ -42,7 +42,7 @@ type Config struct {
 }
 
 func AddConfigFlag(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVarP(&configFile, flagForConfigFile, "c", "", "a toml encoded config file")
+	cmd.Flags().StringVarP(&configFile, flagForConfigFile, "c", "", "a toml encoded config file")
```

Comment:
nit: capitalize "A TOML config file". probably can remove the word "encoded" here.

Reply:
adjusted